### PR TITLE
Synth refix

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

Woops we accidentally removed my one fix to Nagas, and some other stuff was wrongly done + 15 metal per revive? Preposterous.

Also fixes some extra stuff with hardelling cells and charging

## How This Contributes To The Skyrat Roleplay Experience

Better code hopefully

## Proof of Testing

N/A it is almost all the same as before

## Changelog

:cl:
fix: Fixed some erroneous things with the synth update
code: No more harddel cells upon synths charging
/:cl:

### CODE REVIEW

File: code/game/objects/items/tools/weldingtool.dm [hunk 1]
```diff
@@ -154,7 +154,8 @@
 		return NONE
 
 	if (!affecting.get_damage())
-		return
+		balloon_alert(user, "limb not damaged")
+		return ITEM_INTERACT_BLOCKING // SKYRAT FIX
```
Comment (projectkepler-ru) [inline code/game/objects/items/tools/weldingtool.dm]: I'm amazed they fucked that up

**Review by projectkepler-ru — APPROVED** [review]

File: code/game/objects/items/tools/weldingtool.dm (hunk 1 above)
Comment (Maia-J) [reply code/game/objects/items/tools/weldingtool.dm]: There's a bugfix up on tg for it but I like speedrunning it for the sake of less headache for our players

**Comment by Majkl-J** [comment]
Holup someone pointed out something important about this

**Comment by Majkl-J** [comment]
ready to merge

File: code/modules/surgery/organ_manipulation.dm [hunk 2]
```diff
@@ -57,7 +57,7 @@
 /datum/surgery/organ_manipulation/mechanic
 	name = "Hardware Manipulation"
 	requires_bodypart_type = BODYTYPE_ROBOTIC
-	surgery_flags = SURGERY_SELF_OPERABLE | SURGERY_REQUIRE_LIMB
+	surgery_flags = SURGERY_REQUIRE_LIMB // SKYRAT EDIT - Original SURGERY_SELF_OPERABLE | SURGERY_REQUIRE_LIMB
```
Comment (Adrian16199) [inline code/modules/surgery/organ_manipulation.dm]: Sorry to ask but-
Does this make synths not be able to self operate and give themselfs implants alone?
Just for clarification if im reading this right.

**Pull request merged**